### PR TITLE
Editor config, formatting, and copy contract metadata into dest image

### PR DESCRIPTION
**Pull request by epheph**

### CODE REVIEW

File: Dockerfile.parity [hunk 1]
```diff
@@ -19,4 +19,6 @@ WORKDIR /
 FROM keydonix/parity-instantseal
 COPY --from=builder /parity/chains /parity/chains
 
+COPY --from=builder /maker-deployer/output/contracts /contracts
```
Comment (MicahZoltu) [inline Dockerfile.parity:22]: What is the idea here?  Just so that the user can `docker image cp` the original contract out if they wanted?

Comment (MicahZoltu) [inline Dockerfile.parity:22]: Consider also adding the generated TypeScript files for Maker?  Would be nice if they were more generic (not dependent on `eth.js`, but we can address that in the future.

**Review by MicahZoltu — APPROVED** [review]

File: Dockerfile.parity (hunk 1 above)
Comment (epheph) [reply Dockerfile.parity:22]: Exactly, i once had to go view some old output to see what the addresses were, feels right to provide them with the image

Comment (epheph) [reply Dockerfile.parity:22]: Yea, we should make it more exportable

**Pull request merged**